xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -195,14 +264,27 @@
         <Parameter Name="serialNumber" Type="System.Byte[]" />
       </Parameters>
       <Docs>
-        <param name="issuerName">To be added.</param>
-        <param name="generator">To be added.</param>
-        <param name="notBefore">To be added.</param>
-        <param name="notAfter">To be added.</param>
-        <param name="serialNumber">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="issuerName">The X500DistinguishedName for the Issuer.</param>
```

Comment:
I didn't `<see>` the X500DistinguishedName type name here, apparently.  Should it be a link?